elastic-sw-client/upload_diagnostics.py

```diff
@@ -0,0 +1,13 @@
+from elasticsearch import Elasticsearch
+es = Elasticsearch(['elasticsearch.default.svc.cluster.local:9200'])
```

Comment:
Why use a FQDN? This will break demos if they are going to be set up in a different namespace.
Please consider using this
> Elasticsearch(['elasticsearch.{}:9200'.format(os.environ['NAMESPACE'])])